### PR TITLE
Mobile projects

### PR DESCRIPTION
**Pull request by VICTOR-CHUKWU**
a branch of my portfolio site for projects and about section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,65 +1,409 @@
...
+        <h1>Hey There.I’m Victor</h1>
+        <div class="info">I’am a Software Developer</div>
+        <div class="description">
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="socials flex-align-center">
```
Comment (carlos-ssh) [inline index.html:34]: The list of icons uses a `` <ul> `` tag to group them, and `` <li> `` per icon; each icon has a link `` <a> `` tag

File: index.html [hunk 2]
```diff
@@ -1,65 +1,409 @@
...
+                d="M18.4042 1.88042L20 0.337684V0H14.4717L10.5317 9.90989L6.04917 0H0.2525V0.337684L2.11667 2.60547C2.29833 2.77305 2.39333 3.01811 2.36917 3.26484V12.1768C2.42667 12.4977 2.32333 12.8278 2.1 13.0611L0 15.6328V15.9663H5.95417V15.6286L3.85417 13.0611C3.62667 12.8269 3.51917 12.5027 3.565 12.1768V4.46821L8.79167 15.9705H9.39917L13.8933 4.46821V13.6312C13.8933 13.8728 13.8933 13.9225 13.7367 14.0808L12.12 15.6615V16H19.9633V15.6623L18.405 14.1204C18.2683 14.016 18.1975 13.8408 18.2267 13.6707V2.33011C18.1975 2.15916 18.2675 1.984 18.4042 1.88042Z"
+                fill="#DBD8D7"
+              />
+            </svg>
+          </a>
+        </div>
+      </div>
+    </section>
+    <section id="projects" class="mobile-projects">
+      <div class="display-head">My Recent Works</div>
```
Comment (carlos-ssh) [inline index.html:117]: - The title ("My Recent Works") uses a header tag `` <hx> `` (h1, h2, h3 etc.)

File: index.html [hunk 3]
```diff
@@ -1,65 +1,409 @@
...
+          <p class="paragraph">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <div class="anchor-div flex-align-center">
+            <a href="#">css</a>
+            <a href="#">Html</a>
+            <a href="#">Bootstrap</a>
+            <a href="#">Ruby</a>
```
Comment (carlos-ssh) [inline index.html:134]: - Kindly list `` <ul> `` and each of them a  `` <li> `` .

File: index.html [hunk 4]
```diff
@@ -1,65 +1,409 @@
...
+          </div>
+          <button class="btn">See Project</button>
+        </div>
+        <div class="project-one">
+          <div class="project-inner">
+            <div class="display-head-project">
+              Profesional Art Printing Data
```
Comment (carlos-ssh) [inline index.html:141]: - Kindly use text tags.

File: index.html [hunk 5]
```diff
@@ -1,65 +1,409 @@
...
+            </div>
+            <p class="paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <div class="anchor-div flex-align-center">
+              <a class="anchor" href="#">css</a>
+              <a class="anchor" href="#">Html</a>
+              <a class="anchor" href="#">Bootstrap</a>
```
Comment (carlos-ssh) [inline index.html:226]: - The technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li.

File: index.html [hunk 6]
```diff
@@ -1,65 +1,409 @@
...
```
Comment (carlos-ssh) [inline index.html:207]: - The technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li.

File: index.html [hunk 7]
```diff
@@ -1,65 +1,409 @@
...
```
Comment (carlos-ssh) [inline index.html:188]: - The technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li.

File: index.html [hunk 8]
```diff
@@ -1,65 +1,409 @@
...
```
Comment (carlos-ssh) [inline index.html:169]: - The technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @VICTOR-CHUKWU , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Good job with your project!!. Kindly fix the issues displayed below. A few changes are repeated. Keep coding hard!! 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 9]
```diff
@@ -1,65 +1,433 @@
...
+        <div class="info">I’am a Software Developer</div>
+        <div class="description">
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="socials flex-align-center">
+          <a href="#">
```
Comment (mke2111) [inline index.html:35]: - [ ] As previously suggested, each of these icons should be placed in `li` tags that are also grouped by the `ul` tag

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @VICTOR-CHUKWU, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 9 above)
Comment (mke2111) [inline index.html:35]: - [ ] As previously suggested, each of these icons should be placed in `li` tags that are also grouped by the `ul` tag
Something like
ul > li > a

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Victor Chukwu, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**